ecommerce-product-page-main/styled.css

```diff
@@ -200,98 +197,65 @@ body{
     background-color: hsl(26,100%,55%);
 }
 
-
-
-img.space{
-    margin-top: 12px;
-    width: 130px;
-}
-
-ul.list{
-
+.list {
     display: flex;
-    list-style: none;
+    text-decoration: none;
+    gap: 10px;
+    align-items: center;
 }
 
 
-
-ul.list li+li:before{
-
-    padding: 10px;
-    content: " ";
-}       
-
-ul.list li a{
-
+.list a{
     color: hsl(219, 9%, 45%);
     text-decoration: none;
-
 }
 
 
-ul.list li a:hover{
-    padding-bottom: 38px;
+.list a:hover{
     border-bottom: 3px solid hsl(26,100%,55%);
 }
 
 
 .cart{

```

Comment:
This is what i was asking about in my line 234, how did you get be this to be spaced away from the menu links?
cos i used "padding-bottom".